prime-router/src/main/kotlin/cli/tests/TestReportStream.kt

```diff
@@ -1984,4 +1985,95 @@ class OtcProctored : CoolTest() {
             return bad("Tests FAILED: " + failures)
         }
     }
+}
+
+class CrudRestApi : CoolTest() {
+    override val name = "CrudRestApi"
+    override val description = "Testing CRUD REST API"
+    override val status = TestStatus.DRAFT
+
+    val dummyOrgObjString = "{" +
+        "\"name\": \"dummy\"," +
+        "\"description\": \"PRIME's Dummy organization for CRUD API test\"," +
+        "\"jurisdiction\":\"FEDERAL\"," +
+        "\"stateCode\": null," +
+        "\"countyName\": null," +
+        "\"meta\": {" +
+        "\"version\": 0," +
+        "\"createdBy\": \"local@test.com\"," +
+        "\"createdAt\": \"2021-10-16T13:02:22.382422Z\"" +
+        "}" +
+        "}"
+
+    override suspend fun run(environment: ReportStreamEnv, options: CoolTestOptions): Boolean {
+        ugly("Starting CRUD REST API ${environment.urlPrefix}")
+
+        // Make sure dummy-org does not exist.
+        val (getRespCode, getJson) = HttpUtilities.getOrganization(
+            environment,
+            "dummy",
+            simpleRepSender,
+            options.key
+        )
+        echo(getJson)
```

Comment:
Add context to this echo since you have multiple json echos here.  Maybe add another echo with a string stating what the response is for